s-ccs/mne-bids-pipeline
# eyelink sync

- Prepare for 1.4.0 release (#753)
- MAINT: Cleanup dependencies and doc build (#755)
- ENH: Add annotation of head velocity (#757)
- BUG: Fix bug with cache invalidation (#756)
- ENH: Add T1 and FLASH BEM to website (#758)
- BUG: Fix bug with empty room processing (#761)
- ENH: More compact logging (#764)
- ENH: Add eSSS (#762)
- ENH: Cache grand-average steps (#765)
- MAINT: Test MD5 hashing too (#768)
- ENH: Better logging of distances (#769)
- MAINT: pre-commit.ci (#774)
- Added deriv_root argument to CLI (#773)
- MAINT: Use pooch for downloading web data (#775)
- [pre-commit.ci] pre-commit autoupdate (#776)
- API: Change default to _cache (#778)
- [pre-commit.ci] pre-commit autoupdate (#780)
- ENH: Validate inputs with pydantic (#779)
- TST: Rerun CIs (#781)
- BUG: Fix bug where individual invs were not rendered (#782)
- [pre-commit.ci] pre-commit autoupdate (#784)
- [pre-commit.ci] pre-commit autoupdate (#786)
- [pre-commit.ci] pre-commit autoupdate (#787)
- [pre-commit.ci] pre-commit autoupdate (#788)
- [pre-commit.ci] pre-commit autoupdate (#789)
- [pre-commit.ci] pre-commit autoupdate (#790)
- MAINT: Fix bug with item (#791)
- [pre-commit.ci] pre-commit autoupdate (#792)
- Ensure link in  "You're not viewing the latest stable version" banner is readable in light and dark theme (#793)
- [pre-commit.ci] pre-commit autoupdate (#795)
- Reduce logging output when creating and saving reports (#799)
- Use inst.pick() instead of inst.pick_types() (#800)
- Use cleaned epochs for sensor-space decoding (#796)
- [pre-commit.ci] pre-commit autoupdate (#803)
- Add "picard-extended_infomax" ICA algorithm (#801)
- Do not reject epochs based on PTP threshold before ICA cleaning (#806)
- Add `"autoreject_local"` as new option for `preprocessing/ptp_reject` (#807)
- ENH: Table sorting and ds004229 updates (#809)
- [pre-commit.ci] pre-commit autoupdate (#811)
- Add support for `autoreject` (local) before running ICA (#810)
- BUG: Fix bug with verbosity
- Do not interpolate bad segments using autoreject before ICA (#816)
- Omit bad channels when gathering data used for decoding (#817)
- [pre-commit.ci] pre-commit autoupdate (#818)
- Allow decoding using epochs at different stages of pre-processing (#819)
- Prepare for 1.5.0 release (#821)
- FIX: PyQt6 (#822)
- The "Behavior changes" heading was incorrectly commented out in the changelog (#823)
- MAINT: Fix CIs (#824)
- Switch build backend to hatchling (#825)
- [pre-commit.ci] pre-commit autoupdate (#826)
- [pre-commit.ci] pre-commit autoupdate (#827)
- [pre-commit.ci] pre-commit autoupdate (#828)
- [pre-commit.ci] pre-commit autoupdate (#829)
- [pre-commit.ci] pre-commit autoupdate (#830)
- FIX: Paths and pyarrow (#834)
- MAINT: Test result caching (#836)
- MAINT: Enable more ruff rules (#838)
- ENH: Add artifact regression (#837)
- BUG: Fix several bugs (#839)
- [pre-commit.ci] pre-commit autoupdate (#843)
- ENH: Write out raw data and SSP events (#840)
- BUG: Fix bug with Maxwell step when find_noise_channels_meg=False (#847)
- [pre-commit.ci] pre-commit autoupdate (#851)
- BUG: Fix bug with multichannel classification (#853)
- ENH: Better event count rep and noise_cov_method (#854)
- BUG: Fix bug with split epochs and ICA (#855)
- [pre-commit.ci] pre-commit autoupdate (#856)
- DOC: Automatic flowchart (#860)
- DOC: Automatically document settings (#863)
- add image_kwargs to report (#859)
- MAINT: Remove datalad (#867)
- ENH: Split ICA into multiple steps (#865)
- ENH: Document relevant steps (#866)
- ENH: Doc config in sheet (#869)
- Fix a typo in the readme (#871)
- Enable stricter linting (#872)
- Improve logging during cache invalidation (#877)
- Update contributing instructions and remove outdated building instructions (#875)
- Fix handling of `analyze_channels` for EEG data, where some steps (like ERP calculation) would previously fail (#883)
- [pre-commit.ci] pre-commit autoupdate (#885)
- Remove ancient config parameter `shortest_event` (#888)
- Add missing release dates to changelog (#887)
- Explicitly depend on `annotated-types` (#886)
- MAINT: Check that all options are used (#889)
- BUG: Fix bug with CSP computation and Maxwell filter (#890)
- [pre-commit.ci] pre-commit autoupdate (#891)
- MAINT: Print longer traceback on report error (#892)
- Set up Dependabot to automatically keep GitHub Actions up-to-date (#893)
- [dependabot]: Bump actions/setup-python from 4 to 5 (#894)
- [dependabot]: Bump actions/download-artifact from 3 to 4 (#895)
- [dependabot]: Bump actions/upload-artifact from 3 to 4 (#896)
- [dependabot]: Bump codecov/codecov-action from 3 to 4 (#898)
- [dependabot]: Bump actions/checkout from 3 to 4 (#897)
- add n sbs to report (#902)
- [pre-commit.ci] pre-commit autoupdate (#903)
- Drop support for Python 3.9, drop `Optional` and `Union` from type hints (#908)
- Render type annotations in the documentation again (#909)
- MAINT: Ensure input changes cause output changes (#904)
- Add number of subjects to grand-average report (cont'd) (#910)
- Do not show `Annotated` types in configuration options documentation (#911)
- Try to fix documentation deployment (#913)
- Restructure configuration options documentation sections (#914)
- [pre-commit.ci] pre-commit autoupdate (#917)
- enhance documentation of caching, continuation of #914 (#918)
- Improve documentation and config validation of `loose` and `depth` parameters; drop support for `loose=None` (#915)
- change default for info to use for inverse #905 (#919)
- ENH: Streamline ICA reporting (#899)
- [pre-commit.ci] pre-commit autoupdate (#923)
- Support variable time windows for epochs metadata creation (#874)
- Raise a more helpful error message when requested subjects are missing from the dataset (#928)
- [pre-commit.ci] pre-commit autoupdate (#929)
- Fix report section on original raw data (didn't contain bad channels and subject or experimenter name) (#931)
- In the report TOC, don't put the run numbers in quotation marks (#933)
- Rename `ica_ctps_ecg_threshold` to `ica_ecg_threshold` (#935)
- Improve logging output when rejecting epochs; and log number of reconstructed ICA and SSP epochs (#936)
- Save ECG and EOG evokeds created during ICA artifact detection to disk (#938)
- Improve logging during ICA artifact discovery and rejection (#937)
- DOC: Fix typo (#940)
- [pre-commit.ci] pre-commit autoupdate (#942)
- [pre-commit.ci] pre-commit autoupdate (#945)
- MAINT: Cleaner hook into mkdocstrings-python (#943)
- [pre-commit.ci] pre-commit autoupdate (#947)
- remove ref-names from .git_archival.txt (#948)
- initial new step for sync

### Before merging …

- [ ] Changelog has been updated (`docs/source/changes.md`)
